flow/Arena.h

```diff
@@ -129,7 +130,13 @@ struct scalar_traits<Arena> : std::true_type {
 };
 
 struct ArenaBlockRef {
-	ArenaBlock* next;
+	// If alignedBufferSize is not 0, alignedBuffer is valid and must be freed with aligned_free()
+	// Otherwise, next is valid
+	size_t alignedBufferSize;
```

Comment:
Do you think it's worth trying to get away with uint32_t here? That way we could get `sizeof(ArenaBlockRef)` down to 16

Reply:
Definitely.  I was only using `size_t` to match `aligned_alloc(),` which is no longer being used anyway because it turned out to waste 4k of memory for every 4k aligned allocation with standard malloc.